include/RemotePluginClient.h

```diff
@@ -60,8 +60,7 @@ class RemotePluginClient : public RemotePluginBase
 
 	bool processMessage( const message & _m ) override;
 
-	virtual void process( const SampleFrame* _in_buf,
```

Comment:
Why did you remove the `const`? Also, why did you remove the newly introduced `SampleFrame` class?

Note that my review is just incomplete, so maybe I will find out later...

Reply:
I'll see if I can add the const back in.

I made them `float*` because they are not actually `SampleFrame*` - they are `float*` shared memory buffers from `m_audioBuffer` which were being reinterpret-casted to `SampleFrame*` then reinterpret-casted back to `float*` when processing for VSTs. (See farther down in this file where I removed the C casts)

Reply:
Fixed